identity_service/repository/mock_user_repository.py

```diff
@@ -140,12 +164,15 @@ async def update_user(self, user: User) -> None:
 
         Raises
         ------
-        ValueError
+        ValueNotFoundError
             Can't update user with provided data
 
         """
-        index: int = self._users.index(user)
-        self._users[index] = user
+        try:
+            index: int = self._users.index(user)
```

Comment:
wrong way to find index, check pr where ive pinged you

Reply:
wont fix